tests/e2e/onboarding.test.ts

```diff
@@ -20,7 +20,7 @@ test('onboarding with link', async ({ page }) => {
 	const username = faker.internet
 		.userName({ firstName, lastName })
 		.slice(0, 20)
-		.replace(/[^a-z0-9_]/g, '_')
+		.toLowerCase()
```

Comment:
Would this solve the weird email problem?

```suggestion
		.trim()
		.replace(/[^a-z0-9_]/g, '_')
		.toLowerCase()
```